src/components/ProjectChangeRequestListItem/ProjectChangeRequestListItem.tsx

```diff
@@ -50,7 +50,7 @@ export const ProjectChangeRequestListItem = ({
     <Card className={className}>
       <CardContent>
         <Typography
-          variant="body1"
+          variant="subtitle1"
```

Comment:
```
 <p> cannot be under another <p>
```

Reply:
The component prop can be used to change the tag without changing the variant prop's styling.

Reply:
Maybe the ones below get `component="span"` instead.